gaeguli/pipeline.c

```diff
@@ -158,16 +162,34 @@ gaeguli_pipeline_dispose (GObject * object)
         "GaeguliPipeline reference!");
   }
 
+  if (!self->loop && g_main_loop_is_running (self->loop)) {
```

Comment:
Isn't `if (!self->loop) { stop loop }` still inverted?

I think calling `g_main_loop_is_running ()` is redundant here. It should suffice to do 
```
  if (self->loop) {
    g_main_loop_quit (self->loop);
  }
```